src/helpers.ts

```diff
@@ -184,9 +252,23 @@ export function loadOrCreateMinter(
   let filteredMinterContract = IFilteredMinterV2.bind(minterAddress);
 
   // values assigned in contract constructors
-  minter.minterFilter = filteredMinterContract
-    .minterFilterAddress()
-    .toHexString();
+  // @dev safely retrieve value, gracefully handle if it reverts
```

Comment:
note: important to add safety here, because Engine contracts can allowlist any minter, which may or may not conform to the expected interfaces